defaults.go

```diff
@@ -18,7 +18,7 @@ package platforms
 
 // DefaultString returns the default string specifier for the platform.
```

Comment:
nit.. 
 // provided platform specification
```suggestion
// DefaultString returns the default string specifier for the platform, with [PR#6](https://github.com/containerd/platforms/pull/6) the result  
// may now also include the OSVersion from the provided platform specification.
```